### ISSUE TITLE
Bootstrap 4.5.x

### ISSUE THREAD

**Issue by malberts** [ProfessionalWiki/Bootstrap#44]
This is not a bug or a feature request. I'm looking at using the Chameleon skin, and by proxy this extension, on a new wiki (MW 1.35.1). When starting a new project I prefer having the latest versions of dependencies, even if there are no specific requirements from my side.

Is there a _technical_ reason why this extension is still using Bootstrap 4.3? I couldn't find any references to issues with either 4.4 or 4.5. Is it simply a matter of everything is working fine and there have not been any requests for features in the newer versions of Bootstrap?

In other words, if I were to bump the Bootstrap library version to 4.5.3, should I expect known issues? I'm going to try this in any case, but just want to make sure whether I should be aware of anything.

**Comment by JeroenDeDauw** [comment]
There are no know issues with using a more recent version of Twitter Bootstrap. A pull request is welcome. And if you try things out, please let us know how it went.
